themes/agnoster.zsh-theme

```diff
@@ -198,6 +198,18 @@ prompt_virtualenv() {
   fi
 }
 
+# Anaconda: current working conda env
+# To replace the conda modification of PS1 do `conda config --set changeps1 False`
+# or add 'changeps1: False' to your .condarc file.
+prompt_conda() {
+  local conda_env="$CONDA_DEFAULT_ENV"
+  if [[ -n $conda_env ]]; then
+    if [[ -z $CONDA_PROMPT_MODIFIER ]]; then
+      prompt_segment blue black "conda:$conda_env"
```

Comment:
I just learned that `$CONDA_DEFAULT_ENV` might contain the full env path (like `$CONDA_PREFIX`) and not just the env name.

This happens when you create a dedicated env outside of conda's default env location, e.g with command like:
```sh
conda create --prefix /arbitrary/path/to/myenv
```

So in order to prevent the promp to be too long it is better to `basename` it.
```suggestion
      prompt_segment blue black "conda:$(basename $conda_env)"
```

Reply:
An excellent suggestion. I can see how this could easily make the prompt far too long. I think it would be nice to somehow indicate that it is an environment in a non-default location if possible. I cannot think of a nice way to do it though. Perhaps by changing the segment colour to a different colour such as yellow?